lib/src/facts/windows/networking_resolver.cc

```diff
@@ -172,7 +172,13 @@ namespace facter { namespace facts { namespace windows {
                     bool ipv6 = it->Address.lpSockaddr->sa_family == AF_INET6;
 
                     binding b;
-                    b.address = addr;
+
+                    // IpAddress6 contains interface identifier, and should be removed when returning the ipaddress6
+                    if (ipv6){
+                        b.address = addr.substr(0, addr.find('%'));
```

Comment:
you could use the ternary operator here: `b.address = ipv6 ? addr.substr(0, addr.find('%')) : addr;` or similar